### ISSUE TITLE
Cannot craft Control Seal: Block Breaker

### ISSUE THREAD

**Issue by JoelLaviron** [pyure/infitech-prototype#406]
Recipe for Control Seal: Block Breaker doesn't appear on the Thaumonomicon while the corresponding advanced seal is present. In JEI the recipe is listed as infusion but without any additional item. The infusion refuses to start with the empty seal in the central altar and the required essencia.

**Comment by pyure** [comment]
Can you tell if this is a Thaumcraft issue or a problem with the pack specifically?

**Comment by JoelLaviron** [comment]
I'm not sure. I did not find any similar issue reported directly on thaumcraft though. Since the Thaumonomicon is marked as having some recipes modified/disabled in infitech 3, I though I would report here first and it would be forwarded to TC if needed. I will try to check tomorrow if it is affecting TC directly.

**Comment by JoelLaviron** [comment]
In fact it is removed by the script thaumcraft.zs provided in infitech3:
```
mods.thaumcraft.Infusion.removeRecipe(sealBlockBreaker);
mods.thaumcraft.Infusion.registerRecipe(
  "it3_seal_block_breaker", 
  "SEALBREAK", 
  sealBlockBreaker, 
  1, 
  [<aspect:instrumentum> * 10, <aspect:perditio> * 10, <aspect:humanus> * 10], 
  sealBlank, 
  [<ore:toolHeadShovelGold>, <ore:toolHeadAxeGold>, <ore:toolHeadPickaxeGold>]);
  
<thaumcraft:thaumonomicon>.addTooltip(format.aqua("Some recipes may be removed from this tome.  See JEI."));
```
So in fact the original recipee was removed.
I haven't tried yet the infusion with the gold tool heads so I don't know if it works but it is not correctly listed in JEI nor in the Thaumonomicon.

**Comment by pyure** [comment]
ah I see the problem right away

Those gold tool parts no longer exist as of a few GT versions.  I can fix that.

**Comment by pyure** [comment]
Fixed in infitech-prototype-0.42b
